### PR TITLE
[bot] Fingerprints: add missing FW versions from new users

### PR DESCRIPTION
**Pull request by sshane**
## ✅ Updating FW version database with FW from 3 dongles (3 platforms) in last 30 days:
<details open><summary>View table</summary><br>

| Dongle, platform, VIN                                                                                                               | Name from VIN 🆚 CarInfo                                             | Segments             | Bad seg tags   | Good seg tags                                           |
|:------------------------------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------|:---------------------|:---------------|:--------------------------------------------------------|
| [301a36a105223ec3](https://useradmin.comma.ai/?onebox=301a36a105223ec3)<br><sub>TOYOTA COROLLA TSS2 2019<br>JTNK4RBE1........</sub> | TOYOTA Corolla Hatchback 2019 🆚<br>Toyota Corolla Hatchback 2019-22 | 17 routes/7.3 hours  |                | - all lat active: 114<br>- no input all lat active: 69  |
| [eafe068b1de10baf](https://useradmin.comma.ai/?onebox=eafe068b1de10baf)<br><sub>TOYOTA RAV4 2019<br>2T3RWRFV2........</sub>         | TOYOTA RAV4 Hybrid 2021 🆚<br>Toyota RAV4 Hybrid 2019-21             | 6 routes/2.3 hours   |                | - all lat active: 68<br>- no input all lat active: 46   |
| [723b102eb5d8291e](https://useradmin.comma.ai/?onebox=723b102eb5d8291e)<br><sub>TOYOTA PRIUS 2017<br>000000000........</sub>        | None 🆚<br>None                                                      | 56 routes/24.0 hours |                | - all lat active: 654<br>- no input all lat active: 401 |

Dongles to re-run category: `723b102eb5d8291e 301a36a105223ec3 eafe068b1de10baf`
</details>

## ⏳️ 27 dongles (16 platforms) do not yet have enough data to be added:
<details ><summary>View table</summary><br>

| Dongle, platform, VIN                                                                                                               | Name from VIN 🆚 CarInfo                                                                          | Segments              | Bad seg tags                                                                                         | Good seg tags                                         |
|:------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------|:----------------------|:-----------------------------------------------------------------------------------------------------|:------------------------------------------------------|
| [06a5e5d1657e6282](https://useradmin.comma.ai/?onebox=06a5e5d1657e6282)<br><sub>LEXUS IS 2023<br>JTHGZ1E27........</sub>            | LEXUS IS 2023 🆚<br>Lexus IS 2022-23                                                              | 3 routes/0.5 hours    |                                                                                                      | - all lat active: 7<br>- no input all lat active: 3   |
| [d083dc54dae836f0](https://useradmin.comma.ai/?onebox=d083dc54dae836f0)<br><sub>TOYOTA PRIUS TSS2 2021<br>000000000........</sub>   | None 🆚<br>None                                                                                   | 4 routes/1.7 hours    |                                                                                                      | - all lat active: 28<br>- no input all lat active: 13 |
| [8d348ed0b5cb0e07](https://useradmin.comma.ai/?onebox=8d348ed0b5cb0e07)<br><sub>RAM 1500 5TH GEN<br>1C6SRFMT5........</sub>         | RAM 1500 2022 🆚<br>Ram 1500 2019-24                                                              | 19 routes/3.2 hours   | - [VIN spotty (info): 1](https://useradmin.comma.ai/?onebox=8d348ed0b5cb0e07%7C2024-02-25--09-54-04) | - all lat active: 15<br>- no input all lat active: 7  |
| [d51a46432161d94a](https://useradmin.comma.ai/?onebox=d51a46432161d94a)<br><sub>LEXUS GS F 2016<br>JTHBP1BL0........</sub>          | LEXUS GS 2016 🆚<br>Lexus GS F 2016                                                               | 7 routes/2.7 hours    | - [VIN model mismatch (info): 1](https://useradmin.comma.ai/?onebox=d51a46432161d94a)                | - all lat active: 0                                   |
| [3a02fd61e8d654b9](https://useradmin.comma.ai/?onebox=3a02fd61e8d654b9)<br><sub>HONDA ODYSSEY 2018<br>5FNRL6H89........</sub>       | HONDA Odyssey 2020 🆚<br>Honda Odyssey 2018-20                                                    | 47 routes/10.5 hours  |                                                                                                      | - all lat active: 0                                   |
| [319908404e59fb49](https://useradmin.comma.ai/?onebox=319908404e59fb49)<br><sub>TOYOTA PRIUS 2017<br>JTDKBRFU6........</sub>        | TOYOTA Prius 2018 🆚<br>Toyota Prius 2017-20                                                      | 6 routes/3.0 hours    |                                                                                                      | - all lat active: 0                                   |
| [5dccf40c6d8d548e](https://useradmin.comma.ai/?onebox=5dccf40c6d8d548e)<br><sub>HONDA PILOT 2017<br>5FNYF6H38........</sub>         | HONDA Pilot 2021 🆚<br>Honda Pilot 2016-22                                                        | 9 routes/5.3 hours    |                                                                                                      | - all lat active: 33<br>- no input all lat active: 23 |
| [d02240bb0738ae9d](https://useradmin.comma.ai/?onebox=d02240bb0738ae9d)<br><sub>TOYOTA COROLLA TSS2 2019<br>5YFS4RCE8........</sub> | TOYOTA Corolla 2020 🆚<br>Toyota Corolla 2020-22                                                  | 1 routes/0.1 hours    |                                                                                                      | - all lat active: 0                                   |
| [aeeec17f13cb4cef](https://useradmin.comma.ai/?onebox=aeeec17f13cb4cef)<br><sub>TOYOTA RAV4 2019<br>2T3EWRFV9........</sub>         | TOYOTA RAV4 Hybrid 2020 🆚<br>Toyota RAV4 Hybrid 2019-21                                          | 4 routes/0.2 hours    |                                                                                                      | - all lat active: 0                                   |
| [3ac08dbb3c723ffb](https://useradmin.comma.ai/?onebox=3ac08dbb3c723ffb)<br><sub>HONDA PILOT 2017<br>5FNYF5H59........</sub>         | HONDA Pilot 2021 🆚<br>Honda Pilot 2016-22                                                        | 3 routes/0.2 hours    |                                                                                                      | - all lat active: 5<br>- no input all lat active: 1   |
| [cec761a4a881a563](https://useradmin.comma.ai/?onebox=cec761a4a881a563)<br><sub>RAM 1500 5TH GEN<br>1C6SRFHT1........</sub>         | RAM 1500 2019 🆚<br>Ram 1500 2019-24                                                              | 3 routes/0.3 hours    |                                                                                                      | - all lat active: 0                                   |
| [6ca0bbb0b026b5a5](https://useradmin.comma.ai/?onebox=6ca0bbb0b026b5a5)<br><sub>HONDA CR-V HYBRID 2019<br>5J6RT6H87........</sub>   | HONDA CR-V Hybrid 2022 🆚<br>Honda CR-V Hybrid 2017-20<br><sub>🎉 <b>New model year!</b> 🎉</sub> | 12 routes/5.6 hours   |                                                                                                      | - all lat active: 46<br>- no input all lat active: 27 |
| [a6de4da35307d7b8](https://useradmin.comma.ai/?onebox=a6de4da35307d7b8)<br><sub>RAM 1500 5TH GEN<br>1C6SRFHT4........</sub>         | RAM 1500 2022 🆚<br>Ram 1500 2019-24                                                              | 11 routes/3.1 hours   |                                                                                                      | - all lat active: 20                                  |
| [8363f866866f25e7](https://useradmin.comma.ai/?onebox=8363f866866f25e7)<br><sub>HONDA ACCORD 2018<br>1HGCV2F32........</sub>        | HONDA Accord 2022 🆚<br>Honda Accord 2018-22                                                      | 2 routes/0.3 hours    |                                                                                                      | - all lat active: 4<br>- no input all lat active: 1   |
| [9e3afabaed74d30f](https://useradmin.comma.ai/?onebox=9e3afabaed74d30f)<br><sub>TOYOTA C-HR 2018<br>JTNLY3FXX........</sub>         | None 🆚<br>None                                                                                   | 23 routes/4.1 hours   |                                                                                                      | - all lat active: 32<br>- no input all lat active: 8  |
| [030e7d18f7c4ea9d](https://useradmin.comma.ai/?onebox=030e7d18f7c4ea9d)<br><sub>RAM 1500 5TH GEN<br>1C6SRFHTX........</sub>         | RAM 1500 2021 🆚<br>Ram 1500 2019-24                                                              | 105 routes/33.5 hours |                                                                                                      | - all lat active: 2                                   |
| [451be4fe585fdac0](https://useradmin.comma.ai/?onebox=451be4fe585fdac0)<br><sub>TOYOTA PRIUS TSS2 2021<br>000000000........</sub>   | None 🆚<br>None                                                                                   | 4 routes/0.4 hours    |                                                                                                      | - all lat active: 0                                   |
| [b952ae0f9a30d194](https://useradmin.comma.ai/?onebox=b952ae0f9a30d194)<br><sub>MAZDA CX-5 2022<br>JM3KFBXYX........</sub>          | MAZDA CX-5 2022 🆚<br>Mazda CX-5 2022-24                                                          | 2 routes/0.6 hours    |                                                                                                      | - all lat active: 3<br>- no input all lat active: 1   |
| [b38e50aed7b28094](https://useradmin.comma.ai/?onebox=b38e50aed7b28094)<br><sub>LEXUS ES 2018<br>JTHBW1GG2........</sub>            | LEXUS ES Hybrid 2017 🆚<br>Lexus ES Hybrid 2017-18                                                | 15 routes/5.1 hours   |                                                                                                      | - all lat active: 50<br>- no input all lat active: 27 |
| [a86f8c298cdf3887](https://useradmin.comma.ai/?onebox=a86f8c298cdf3887)<br><sub>LEXUS NX 2018<br>JTJBARBZ9........</sub>            | LEXUS NX 2019 🆚<br>Lexus NX 2018-19                                                              | 55 routes/10.1 hours  |                                                                                                      | - all lat active: 30<br>- no input all lat active: 21 |
| [0662797832aac2eb](https://useradmin.comma.ai/?onebox=0662797832aac2eb)<br><sub>TOYOTA COROLLA TSS2 2019<br>JTHU95BH8........</sub> | None 🆚<br>None                                                                                   | 10 routes/3.7 hours   |                                                                                                      | - all lat active: 35<br>- no input all lat active: 26 |
| [2ca288350c66ae18](https://useradmin.comma.ai/?onebox=2ca288350c66ae18)<br><sub>JEEP GRAND CHEROKEE 2019<br>1C4RJFJT7........</sub> | JEEP Grand Cherokee 2020 🆚<br>Jeep Grand Cherokee 2019-21                                        | 4 routes/1.4 hours    |                                                                                                      | - all lat active: 15<br>- no input all lat active: 7  |
| [22dbdf2ca2bce921](https://useradmin.comma.ai/?onebox=22dbdf2ca2bce921)<br><sub>HONDA PILOT 2017<br>5FNYF6H5X........</sub>         | HONDA Pilot 2021 🆚<br>Honda Pilot 2016-22                                                        | 280 routes/40.2 hours |                                                                                                      | - all lat active: 16<br>- no input all lat active: 2  |
| [c04a3b5508dc7bc3](https://useradmin.comma.ai/?onebox=c04a3b5508dc7bc3)<br><sub>HONDA ACCORD 2018<br>1HGCV2F90........</sub>        | HONDA Accord 2021 🆚<br>Honda Accord 2018-22                                                      | 37 routes/7.5 hours   |                                                                                                      | - all lat active: 8<br>- no input all lat active: 5   |
| [9bac7575fac62395](https://useradmin.comma.ai/?onebox=9bac7575fac62395)<br><sub>HONDA PILOT 2017<br>5FNYF6H93........</sub>         | HONDA Pilot 2018 🆚<br>Honda Pilot 2016-22                                                        | 143 routes/24.8 hours |                                                                                                      | - all lat active: 214<br>- no input all lat active: 9 |
| [2653b4b309780e1a](https://useradmin.comma.ai/?onebox=2653b4b309780e1a)<br><sub>TOYOTA COROLLA TSS2 2019<br>000000000........</sub> | None 🆚<br>None                                                                                   | 1 routes/0.0 hours    |                                                                                                      | - all lat active: 0                                   |
| [7de93c2ac463075e](https://useradmin.comma.ai/?onebox=7de93c2ac463075e)<br><sub>HONDA ACCORD 2018<br>1HGCV1F40........</sub>        | HONDA Accord 2021 🆚<br>Honda Accord 2018-22                                                      | 5 routes/0.7 hours    |                                                                                                      | - all lat active: 1<br>- no input all lat active: 1   |

Dongles to re-run category: `aeeec17f13cb4cef 6ca0bbb0b026b5a5 a6de4da35307d7b8 b38e50aed7b28094 319908404e59fb49 d02240bb0738ae9d d083dc54dae836f0 a86f8c298cdf3887 8363f866866f25e7 0662797832aac2eb cec761a4a881a563 2ca288350c66ae18 06a5e5d1657e6282 9e3afabaed74d30f 5dccf40c6d8d548e 030e7d18f7c4ea9d 8d348ed0b5cb0e07 22dbdf2ca2bce921 d51a46432161d94a 2653b4b309780e1a 451be4fe585fdac0 3a02fd61e8d654b9 c04a3b5508dc7bc3 3ac08dbb3c723ffb 9bac7575fac62395 7de93c2ac463075e b952ae0f9a30d194`
</details>

## ⚠️ 13 dongles (7 platforms) have issues that need to be resolved before adding:
<details ><summary>View table</summary><br>

| Dongle, platform, VIN                                                                                                             | Name from VIN 🆚 CarInfo                                             | Segments              | Bad seg tags                                                                                                                                                                                                                                                                                                    | Good seg tags                                           |
|:----------------------------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------|:----------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--------------------------------------------------------|
| [fc4d8f4e5f8b353a](https://useradmin.comma.ai/?onebox=fc4d8f4e5f8b353a)<br><sub>HONDA CIVIC 2022<br>19XFL1H88........</sub>       | HONDA Civic Hatchback 2023 🆚<br>Honda Civic Hatchback 2022-23       | 103 routes/28.6 hours | - [missing ECU FW: 742](https://useradmin.comma.ai/?onebox=fc4d8f4e5f8b353a%7C2024-02-12--16-05-04)<br>- [spotty FW: 742](https://useradmin.comma.ai/?onebox=fc4d8f4e5f8b353a%7C2024-02-12--16-05-04)                                                                                                           | - all lat active: 390<br>- no input all lat active: 265 |
| [61c374c8e5e59db3](https://useradmin.comma.ai/?onebox=61c374c8e5e59db3)<br><sub>HONDA CIVIC 2022<br>19XFL2H89........</sub>       | HONDA Civic Hatchback 2023 🆚<br>Honda Civic Hatchback 2022-23       | 105 routes/33.1 hours | - [missing ECU FW: 1343](https://useradmin.comma.ai/?onebox=61c374c8e5e59db3%7C2024-02-05--14-25-17)<br>- [spotty FW: 645](https://useradmin.comma.ai/?onebox=61c374c8e5e59db3%7C2024-02-14--15-28-59)<br>- [FW not exact match: 1](https://useradmin.comma.ai/?onebox=61c374c8e5e59db3%7C2024-02-05--14-25-17) | - all lat active: 308<br>- no input all lat active: 224 |
| [2024aefc4c4f766c](https://useradmin.comma.ai/?onebox=2024aefc4c4f766c)<br><sub>MAZDA CX-9 2021<br>JM3TCBAYX........</sub>        | MAZDA CX-9 2023 🆚<br>Mazda CX-9 2021-23                             | 75 routes/26.8 hours  | - [car faulted: 2](https://useradmin.comma.ai/?onebox=2024aefc4c4f766c%7C2024-02-18--15-54-11)                                                                                                                                                                                                                  | - all lat active: 577<br>- no input all lat active: 309 |
| [1ef3d03831782308](https://useradmin.comma.ai/?onebox=1ef3d03831782308)<br><sub>TOYOTA PRIUS 2017<br>JTDKARFP2........</sub>      | TOYOTA Prius Prime 2017 🆚<br>Toyota Prius Prime 2017-20             | 10 routes/1.4 hours   | - [spotty FW: 61](https://useradmin.comma.ai/?onebox=1ef3d03831782308%7C2024-02-25--13-41-46)<br>- [VIN spotty (info): 1](https://useradmin.comma.ai/?onebox=1ef3d03831782308%7C2024-02-26--05-22-31)                                                                                                           | - all lat active: 6<br>- no input all lat active: 2     |
| [42c4ff7fee7afa9d](https://useradmin.comma.ai/?onebox=42c4ff7fee7afa9d)<br><sub>HONDA CIVIC 2022<br>2HGFE2F28........</sub>       | HONDA Civic 2023 🆚<br>Honda Civic 2022-23                           | 4 routes/0.5 hours    | - [missing ECU FW: 28](https://useradmin.comma.ai/?onebox=42c4ff7fee7afa9d%7C2024-02-01--10-15-09)<br>- [FW not exact match: 1](https://useradmin.comma.ai/?onebox=42c4ff7fee7afa9d%7C2024-02-01--10-15-09)                                                                                                     | - all lat active: 5<br>- no input all lat active: 1     |
| [b53e882f1a5c30ae](https://useradmin.comma.ai/?onebox=b53e882f1a5c30ae)<br><sub>LEXUS IS 2018<br>JTHBA1D23........</sub>          | LEXUS IS 2018 🆚<br>Lexus IS 2017-19                                 | 33 routes/6.4 hours   | - [missing ECU FW: 2](https://useradmin.comma.ai/?onebox=b53e882f1a5c30ae%7C2024-02-05--17-13-05)<br>- [spotty FW: 2](https://useradmin.comma.ai/?onebox=b53e882f1a5c30ae%7C2024-02-05--17-13-05)<br>- [VIN spotty (info): 1](https://useradmin.comma.ai/?onebox=b53e882f1a5c30ae%7C2024-01-30--18-27-08)       |                                                         |
| [6e12554fbd20d6d1](https://useradmin.comma.ai/?onebox=6e12554fbd20d6d1)<br><sub>HONDA CIVIC 2022<br>2HGFE1F98........</sub>       | HONDA Civic 2022 🆚<br>Honda Civic 2022-23                           | 236 routes/50.4 hours | - [missing ECU FW: 1633](https://useradmin.comma.ai/?onebox=6e12554fbd20d6d1%7C2024-02-04--19-40-21)<br>- [spotty FW: 1633](https://useradmin.comma.ai/?onebox=6e12554fbd20d6d1%7C2024-02-04--19-40-21)                                                                                                         | - all lat active: 321<br>- no input all lat active: 182 |
| [cee1f89fd5ca2190](https://useradmin.comma.ai/?onebox=cee1f89fd5ca2190)<br><sub>HONDA PILOT 2017<br>5FNYF5H99........</sub>       | HONDA Pilot 2016 🆚<br>Honda Pilot 2016-22                           | 51 routes/10.4 hours  | - [CAN invalid: 5](https://useradmin.comma.ai/?onebox=cee1f89fd5ca2190%7C2024-02-10--13-51-56)                                                                                                                                                                                                                  |                                                         |
| [790069c4b2f63926](https://useradmin.comma.ai/?onebox=790069c4b2f63926)<br><sub>TOYOTA PRIUS 2017<br>JTDKARFU5........</sub>      | TOYOTA Prius 2016 🆚<br>Toyota Prius 2016                            | 4 routes/2.0 hours    | - [missing ECU FW: 48](https://useradmin.comma.ai/?onebox=790069c4b2f63926%7C2024-02-17--20-47-00)<br>- [spotty FW: 48](https://useradmin.comma.ai/?onebox=790069c4b2f63926%7C2024-02-17--20-47-00)<br>- [VIN spotty (info): 1](https://useradmin.comma.ai/?onebox=790069c4b2f63926%7C2024-02-17--20-47-00)     | - all lat active: 1<br>- no input all lat active: 1     |
| [42e3e02718bff70a](https://useradmin.comma.ai/?onebox=42e3e02718bff70a)<br><sub>HONDA CIVIC 2022<br>2HGFE2F59........</sub>       | HONDA Civic 2023 🆚<br>Honda Civic 2022-23                           | 5 routes/1.1 hours    | - [missing ECU FW: 60](https://useradmin.comma.ai/?onebox=42e3e02718bff70a%7C2024-02-14--15-05-37)<br>- [spotty FW: 60](https://useradmin.comma.ai/?onebox=42e3e02718bff70a%7C2024-02-14--15-05-37)                                                                                                             | - all lat active: 8<br>- no input all lat active: 3     |
| [698b51bf70bd1fa8](https://useradmin.comma.ai/?onebox=698b51bf70bd1fa8)<br><sub>HONDA CIVIC 2022<br>19XFL1H80........</sub>       | HONDA Civic Hatchback 2022 🆚<br>Honda Civic Hatchback 2022-23       | 109 routes/33.5 hours | - [missing ECU FW: 1877](https://useradmin.comma.ai/?onebox=698b51bf70bd1fa8%7C2024-02-06--21-15-29)<br>- [spotty FW: 136](https://useradmin.comma.ai/?onebox=698b51bf70bd1fa8%7C2024-02-12--08-32-19)<br>- [FW not exact match: 1](https://useradmin.comma.ai/?onebox=698b51bf70bd1fa8%7C2024-02-06--21-15-29) | - all lat active: 5<br>- no input all lat active: 2     |
| [443fc6ac84d80c7e](https://useradmin.comma.ai/?onebox=443fc6ac84d80c7e)<br><sub>TOYOTA HIGHLANDER 2017<br>5TDBGRFH8........</sub> | TOYOTA Highlander Hybrid 2018 🆚<br>Toyota Highlander Hybrid 2017-19 | 2 routes/0.4 hours    | - [missing ECU FW: 22](https://useradmin.comma.ai/?onebox=443fc6ac84d80c7e%7C2024-02-21--06-35-30)<br>- [FW not exact match: 1](https://useradmin.comma.ai/?onebox=443fc6ac84d80c7e%7C2024-02-21--06-35-30)                                                                                                     |                                                         |
| [75d1c547e2d4725b](https://useradmin.comma.ai/?onebox=75d1c547e2d4725b)<br><sub>HONDA ODYSSEY 2018<br>5FNRL6H91........</sub>     | HONDA Odyssey 2019 🆚<br>Honda Odyssey 2018-20                       | 33 routes/9.1 hours   | - [missing ECU FW: 7](https://useradmin.comma.ai/?onebox=75d1c547e2d4725b%7C2024-02-15--08-03-18)<br>- [spotty FW: 7](https://useradmin.comma.ai/?onebox=75d1c547e2d4725b%7C2024-02-15--08-03-18)                                                                                                               | - all lat active: 91<br>- no input all lat active: 68   |

Dongles to re-run category: `42c4ff7fee7afa9d 1ef3d03831782308 fc4d8f4e5f8b353a 61c374c8e5e59db3 6e12554fbd20d6d1 790069c4b2f63926 443fc6ac84d80c7e 75d1c547e2d4725b 698b51bf70bd1fa8 b53e882f1a5c30ae 42e3e02718bff70a cee1f89fd5ca2190 2024aefc4c4f766c`
</details>

### CODE REVIEW

**Comment by sshane** [comment]
token expired!

**Comment by adeebshihadeh** [comment]
This is a lot of new FW, so converted to draft. @sshane I assume you'll want to look into all of these first?

File: selfdrive/car/toyota/fingerprints.py [hunk 1]
```diff
@@ -823,6 +824,7 @@
       b'\x03896634760100\x00\x00\x00\x008966A4703000\x00\x00\x00\x00897CF4701003\x00\x00\x00\x00',
       b'\x03896634760200\x00\x00\x00\x008966A4703000\x00\x00\x00\x00897CF4701003\x00\x00\x00\x00',
       b'\x03896634760200\x00\x00\x00\x008966A4703000\x00\x00\x00\x00897CF4701004\x00\x00\x00\x00',
+      b'\x03896634760200\x00\x00\x00\x008966A4703000\x00\x00\x00\x00????????????????',
```
Comment (sshane) [inline selfdrive/car/toyota/fingerprints.py:827]: this is odd

Comment (sshane) [reply selfdrive/car/toyota/fingerprints.py:827]: Previous version used to be `b'\x03896634760200\x00\x00\x00\x008966A4703000\x00\x00\x00\x00897CF4701003\x00\x00\x00\x00'`, and the user's EPS version bumped slightly at the same time causing it to fuzzy fingerprint: `b'8965B47023\x00\x00\x00\x00\x00\x00'` -> `b'8965B47070\x00\x00\x00\x00\x00\x00'` + the VIN stopped returning.

So maybe a bad ECU update from a dealer? The strange part is that the user is not at any kind of auto shop between these two routes as far as I can tell

723b102eb5d8291e/2024-02-21--11-05-51
723b102eb5d8291e/2024-02-21--15-22-57

Comment (sshane) [reply selfdrive/car/toyota/fingerprints.py]: The Prius seems to have recovered. Will add the check next time we see this.

**Pull request merged**